README.md

```diff
@@ -1 +1,77 @@
-# ruby-capstone
\ No newline at end of file
+## OOP school library
+
+This app is develped in ruby and allows users to:
+
+- Ruby console app with an user interface.
+- You can add books or music albums, and you can create a label or genre for the said item.
+- The application preserves user data by using JSON files while exiting the application. 
+
+## Built With
+
+- Ruby
+- OOP
+
+### Technologies used
+
+- Github
+- GitFlow
+- Visual Studio Code
+- rvm
+- Rspec
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- go to the directory that you want to clone the repository in.
+- right click and choose Git Bash here.
+- type "git clone https://github.com/ShoiraTa/ruby-capstone.git".
+- go to the repository folder in your command prompt.
+- run bundle install.
+
+or
+
+- Go to the dist folder and open index.html
+
```

Comment:
- [ ] Please write the correct steps to run the project locally. Currently, we don't have `an index.html` file to run. Instead, the command should be `ruby main.rb`

- [ ] Also include the command to test the project. [Hint: `rspec`]

Reply:
Thank you, we updated readme file